### PR TITLE
SUS-1929

### PR DESCRIPTION
**Pull request by timmyquivy**
Removing CloseMyAccount supported langauge conditional - After discussion, the behavior, even for non-support languages, is always better going through CloseMyAccount as opposed to going through Special:Contact.

### CODE REVIEW

File: extensions/wikia/SpecialContact2/SpecialContact.body.php [hunk 1]
```diff
@@ -762,16 +753,3 @@ private function validateUserName( $userName ) {
...
-	 *
-	 * @return boolean True if CloseMyAccount is enabled and supported in the
-	 *                 current language, false otherwise
-	 */
-	private function isCloseMyAccountSupported() {
-		global $wgContLang, $wgEnableCloseMyAccountExt, $wgSupportedCloseMyAccountLang;
-		return !empty( $wgEnableCloseMyAccountExt )
-				&& in_array( $wgContLang->getCode(), $wgSupportedCloseMyAccountLang );
-	}
-}
```
Comment (Grunny) [inline extensions/wikia/SpecialContact2/SpecialContact.body.php]: One brace too many removed, will cause a fatal error.

File: extensions/wikia/SpecialContact2/SpecialContact.body.php [hunk 2]
```diff
@@ -762,16 +753,3 @@ private function validateUserName( $userName ) {
...
-	 * Check if the CloseMyAccount extension is enabled and supported in the
-	 * current language.
-	 *
-	 * @return boolean True if CloseMyAccount is enabled and supported in the
-	 *                 current language, false otherwise
-	 */
-	private function isCloseMyAccountSupported() {
-		global $wgContLang, $wgEnableCloseMyAccountExt, $wgSupportedCloseMyAccountLang;
-		return !empty( $wgEnableCloseMyAccountExt )
-				&& in_array( $wgContLang->getCode(), $wgSupportedCloseMyAccountLang );
```
Comment (Grunny) [inline extensions/wikia/SpecialContact2/SpecialContact.body.php:775]: Can we remove `wgSupportedCloseMyAccountLang` from the config repo as well as cleanup?

**Review by Grunny — CHANGES_REQUESTED** [review]

**Comment by timmyquivy** [comment]
Fixed the bracket, don't know what happened with my GUI there. Looking at config.

**Comment by timmyquivy** [comment]
@Grunny: https://github.com/Wikia/config/pull/2100. I believe we are now good?

File: extensions/wikia/SpecialContact2/SpecialContact.body.php (hunk 2 above)
Comment (timmyquivy) [reply extensions/wikia/SpecialContact2/SpecialContact.body.php:775]: This is done. https://github.com/Wikia/config/pull/2100

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@timmyquivy - I have made a comment under the ticket https://wikia-inc.atlassian.net/browse/SUS-1929

**Pull request closed without merging**